Project03.py

```diff
@@ -17,7 +17,7 @@ def climatedata(path):
 
 
 
-
+# I think you didn't save your changes before pushing them to GitHub.
 
 if __name__ == "__main__":
     mainpath = r"C:\Users\griff\OneDrive\718\raw data\climate.csv" 
```

Comment:
Put you data in the GitHub, if it's not too large. [Absolute path](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExanFoMGJqODRsZHVlb2dqMzI4cjcwY3NjaDEweGh1ajhzZmdjcjlyOSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/uNgUzhakqXkyI/giphy.gif)s are not a good idea, it would only run for me.